core/app/models/spree/store_credit_category.rb

```diff
@@ -1,6 +1,6 @@
 class Spree::StoreCreditCategory < Spree::Base
-  class_attribute :non_expiring_credit_types
-  self.non_expiring_credit_types = [Spree.t("store_credit.non_expiring")]
+  class_attribute :non_expiring_credit_categories
```

Comment:
Name change is fine by me, but we need to old method to be deprecated before removal.